helm/cluster-cloud-director/values.schema.json

```diff
@@ -523,43 +642,53 @@
             "type": "array"
         },
         "proxy": {
+            "description": "Whether/how outgoing traffic is routed through proxy servers.",
             "properties": {
                 "enabled": {
+                    "title": "Enable",
                     "type": "boolean"
                 },
                 "secretName": {
+                    "title": "TODO",
```

Comment:
What is `secretName`? What's in that secret?

Reply:
This is the name of the secret where the http_proxy, https_proxy and noproxy environment variables are stored. Used to configure containerd.